DEVELOPER.md

```diff
@@ -155,7 +155,46 @@ To deploy a tarball run:
 The tarball can be found in directory `node/target/universal/`
 
 #### Running
-To run rnode locally from within sbt use the revolver plugin. It will start the app in a forked JVM.
+To run RNode locally, first execute:
+```
+~/rchain$ sbt stage
+```
+This will build an executable on the path `./node/target/universal/stage/bin/rnode` relative to the repository.
+
+Next, make a directory where you would like to store the logs, keys, and other information related to your rnode testing:\
+e.x.
+```
+~$ mkdir testing
+```
+
+Inside this directory, create a new directory named `genesis` and add to `genesis` the files `bonds.txt` and `wallets.txt`:
+```
+~/testing$ mkdir genesis
+~/testing$ cd genesis
+~/testing/genesis$ touch wallets.txt
+~/testing/genesis$ touch bonds.txt
+```
+
+Populate `bonds.txt` with the following information: 
+```
+04688b8920885b3d41d517a5a5d3088ae4b801938855e5cb215cfc9bb4e857bde1f5c559d57a8332be1308a282b69e542184ee8231e75f8287a3aa8fe91d5293c4 320000000
+040ee2bbd611a0d630d04574ba9d4b11b05e84d834b0e918d1445ae0d4e6152cfe1b4ea8daeadd8e212d66ad4bb312d4e236b9b0a187326ea40d17a7d137934f62 750000000
+042be28cef3700a6f5d95ec670af061ec69002efee4d553e398b032ea1f5cb9550c262ef06a46f9cdec65660a66d05dd27cbb84e8e714d84822523395ee7cc3790 480000000
```

Comment:
Bonds file contains public keys and without corresponding private keys node cannot be started in validator mode.
I'm not sure if is it good idea to add fixed private keys on the main repo.

Also node has CLI command to generate private/public key.
https://github.com/rchain/rchain/blob/cafbda84e/node/src/main/scala/coop/rchain/node/Main.scala#L172